parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -196,6 +244,16 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 			}
 		}
 
+		// Allow to change dedicated storage items (called by governance-like)
+		match call {
+			RuntimeCall::System(frame_system::Call::set_storage { items })
+				if items.iter().any(|(k, _)| {
+					k.eq(&AllowMultiAssetPools::key()) | k.eq(&LiquidityWithdrawalFee::key())
+				}) =>
+				return true,
+			_ => (),
+		};
```

Comment:
I don't think we need this here, it was put in Westend to manage the migration of bad keys.